### PR TITLE
No longer write archive lists COM

### LINKED ISSUE #760: Archive lists should not be written to COM
**Description**
The archive job is currently writing the lists of files to archive to COM. These are temporary files that should not be written to COM at all; they should use $DATA and be deleted after the archive action completes.

**Requirements**
An archive job that uses $DATA instead of COM, and deletes $DATA after successful completion.

**Dependencies**
None

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
# Description

This PR address issue #760. The only changes contained in this PR are pointing the respective `.txt` files used for the HPSS archive to the `DATA` path. Each defined text file has been updated to explicitly write to `DATA` rather than `COM` such that the files are deleted rather than being archived. 

Resolves #760 

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
This PR was tested by run the local archiving step. The `.txt` files and the respective tarballs are created and written to the expected paths beneath `${DATA}`.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
Didn't run the test, but the code changes look good to me.
Using `$DATA` to stage temporary `txt` files for archiving looks correct.

File: scripts/exglobal_archive.sh
Comment (WalterKolczynski-NOAA) [inline scripts/exglobal_archive.sh]: The jjob header already creates the directory, so doing that again isn't necessary. It also `cd`s into the directory, but it doesn't hurt to make sure of it here.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Looks good other than this one thing.

File: scripts/exglobal_archive.sh
Comment (HenryRWinterbottom) [reply scripts/exglobal_archive.sh]: Done.

File: scripts/exglobal_archive.sh
Comment (KateFriedman-NOAA) [inline scripts/exglobal_archive.sh]: We don't want this line here (`[[ -d ${DATA} ]] && rm -rf "${DATA}"`) since it will remove the `${DATA}` folder created by `ush/jjob_header.sh` https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/jjob_header.sh#L66:L74).

At this point in the job we only need to `cd` into `${DATA}` (what happens on line 159 - `cd "${DATA}" || exit 2`) so let's remove line 158.

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]
Looks good....just need to remove a line in `scripts/exglobal_archive.sh` and see CI testing work. Thanks for working on this @HenryWinterbottom-NOAA!

File: scripts/exglobal_archive.sh
Comment (HenryRWinterbottom) [reply scripts/exglobal_archive.sh]: Done.

Comment (KateFriedman-NOAA) [reply scripts/exglobal_archive.sh]: Thanks @HenryWinterbottom-NOAA !

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks @HenryWinterbottom-NOAA ! Approve pending the successful completion of any CI testing.

**Comment by HenryRWinterbottom** [comment]
> Looks good, thanks @HenryWinterbottom-NOAA ! Approve pending the successful completion of any CI testing.

@KateFriedman-NOAA You are welcome.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending successful CI test completion.

File: ush/hpssarch_gen.sh
Comment (WalterKolczynski-NOAA) [inline ush/hpssarch_gen.sh:1]: Missed one of the new ones on L476 after you merged develop.

Comment (HenryRWinterbottom) [reply ush/hpssarch_gen.sh:1]: Thanks. Done.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**
